test/GogeDao.t.sol

```diff
@@ -176,28 +203,26 @@ contract DaoTest is Utility, Test {
 
         // Verify tokens were sent from Joe to Dao
         assertEq(gogeToken.balanceOf(address(joe)), 0);
-        assertEq(gogeToken.balanceOf(address(gogeDao)), joe_votes);
+        assertEq(gogeToken.balanceOf(address(gogeDao)), joe_votes + gogeDao.minAuthorBal());
 
         // Post-state check.
         assertEq(gogeDao.polls(1, address(joe)), joe_votes);
-        assertEq(gogeDao.totalVotes(1), joe_votes);
-        assertEq(gogeDao.historicalTally(1), joe_votes);
+        assertEq(gogeDao.totalVotes(1), joe_votes + gogeDao.minAuthorBal());
 
         // Verify quorum
-        uint256 num = (gogeDao.totalVotes(1) * 100) /
-            gogeToken.getCirculatingMinusReserve(); // => 10%
+        uint256 num = gogeDao.getProportion(1); // => 10%
         emit log_uint(num);
     }
 
```

Comment:
I want a test(s) that includes having a lot of different polls going on at the same time. At least three simultaneous polls. Not only that but a test(s) to see how the quorum functionality works in the event of multiple polls receiving votes etc. and meeting quorum. There are a lot of unknowns that have not been reached with regard to testing polls and their state changes..

Reply:
Will work on adding this

Reply:
There is already test_gogeDao_removeVotesFromPoll() which tests the addition and removal of votes from 3 simultaneous polls going on. Let's brainstorm how we can create a test with multiple ongoing polls and many advocates and some polls passing, some not, etc.